mlrun/api/db/sqldb/models.py

```diff
@@ -113,10 +114,13 @@ class Tag(Base, BaseModel):
         )
 
         id = Column(Integer, primary_key=True)
-        project = Column(String)
-        name = Column(String)
+        project = Column(String(255, collation=SQLCollationUtil.collation()))
+        name = Column(String(255, collation=SQLCollationUtil.collation()))
         obj_id = Column(Integer, ForeignKey(f"{table}.id"))
-        obj_name = Column(String, ForeignKey(f"{table}.name"))
+        obj_name = Column(
+            String(255, collation=SQLCollationUtil.collation()),
+            ForeignKey(f"{table}.name"),
```

Comment:
I remember I had a problem with `ForeignKey` pointing on a column that is not primary key or has unique constraint, I see in my PR I removed the foreign key from this column, maybe we should do that ?

Reply:
If you remember this is impossible...
The ForeignKey constraint created by alembic has no name, so if we remove it, the alembic migration breaks. The only way to fix this is to rebuild the database from scratch somehow...